### DISCUSSION TITLE
Change color for the about command

### DISCUSSION THREAD

**Discussion by mdicey** [jagrosh/MusicBot#1206] [Questions]
Hello is there anyway for me to change the green color of the about command , the only green color of the command .Thank you
![change color](https://user-images.githubusercontent.com/97178489/187921373-bab078b5-e321-463f-b066-4386ddcadb65.JPG)

**Comment by Valtrexon** [comment]
Good idea! Changing the color can make your design or sketch look more lively and unique. You could check out [sonic coloring sheets](https://sketchytones.com/sonic-coloring-pages/) for some cool color inspiration and creative combos.
